qr-code-auth/README.md

```diff
@@ -0,0 +1,286 @@
+| [PATCH /qr-code-auth](#patch-qr-code-authuser_iduser_idis_authorizedis_authorized) | Updates the is_authorized field of an existing qr-code-auth document for the specified user. |
+
+## POST /qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **device_info** (required, string): Specifies the device information associated with the authentication authentication.
+    - **is_authorized** (required, boolean): Indicates whether the authentication document is authorized or not.
+    - **access_token** (required, string): Specifies the access token associated with the authentication document.
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 201
+    - **Content:**
+      ```json
+      {
+        "message": "String",
+        "data": {
+          "user_id": "String",
+          "device_info": "String",
+          "is_authorized": "Boolean",
+          "access_token": "String"
+        }
+      }
+      ```
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 401
+    - **Content:**
+      ```json
+      {
+        "statusCode": 401,
+        "error": "Unauthorized",
+        "message": "Unauthenticated User."
+      }
+      ```
+  - **Code:** 409
+    - **Content:**
+      ```json
+      {
+        "message": "The authentication document has already been created"
+      }
+      ```
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example for user authentication document creation request:**
+  POST /qr-code-auth<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
+
+  ```json
+  {
+    "device_info": "t5k77PHnuDSrgEzvMJAj",
+    "is_authorized": true,
+    "access_token": "NLFSj7Kz30oHgolfIZtJ"
+  }
+  ```
+
+  Response :
+  Status 201<br/>
+  Content-Type: application/json<br/>
+
+  ```json
+  {
+    "data": {
+      "user_id": "SooJK37gzjIZfFNH0tlL",
+      "device_info": "t5k77PHnuDSrgEzvMJAj",
+      "is_authorized": true,
+      "access_token": "NLFSj7Kz30oHgolfIZtJ"
+    },
+    "message": "authentication document created successfully."
+  }
+  ```
+
+## GET /qr-code-auth
+
+Retrieves THE authentication document.
+
+- **Params**
+  None
+- **Query**
+
+  - user_id : Specifies the ID of the User whose authentication document will be retrieved
+
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  None
+- **Success Response:**
+
+  - **Code:** 200
+
+    - **Content:**
+
+    ```json
+    {
+      "message": "String",
+      "data": {
+        "user_id": "String",
+        "device_info": "String",
```

Comment:
@shreya-mishra  Assuming this call will be made by mobile to get auth info,
device_info does not make sense to be included here. Could you please clarify this as well?

Reply:
Since this function will be used on both the backend and mobile platform, we are sending all the data contained in the document.

Reply:
Thank you for clarifying.